src/pooling/problem_description.cpp

```diff
@@ -79,10 +79,11 @@ NetworkConfig ProblemDescription::MakeNetworkConfig() const
                     std::min(16, std::max(1, prePow2(_out_pix_tile1 * pooling.strides[0])));
 
             int _grp_tile0 = out_width <= 8 ? 8 : (out_width % 32 <= 16 ? 16 : 32);
-            int _grp_tile1 =
-                out_height <= 8
-                    ? 8
-                    : out_height < 16 ? 16 : out_height <= 32 ? 32 : out_height <= 64 ? 64 : 128;
+            int _grp_tile1 = out_height <= 8    ? 8
+                             : out_height < 16  ? 16
```

Comment:
[informative] This looks strange. Tracked in https://github.com/ROCmSoftwarePlatform/MIOpen/pull/1284#pullrequestreview-1127395599.